<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminVerLaundry</class>
 <widget class="QDialog" name="AdminVerLaundry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>                                               BENNETT LAUNDRY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>30</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>                                                                                      VERIFY LAUNDRY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>711</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Enroll. No. :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEnrollNoVer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>BagNo. :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBagNoVer">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Date :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDateVer">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>99</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>---------------------------------------------------------------------------------------------------------------------------------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Kurta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Pajama</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Shirt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>T-Shirt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Pant</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Lower</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Shorts</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Bedsheet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>238</x>
     <y>220</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Pillow Cover</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>238</x>
     <y>250</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Towel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtKurtaVer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPajamaVer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtShirtVer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTShirtVer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPantVer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLowerVer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtShortsVer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBedsheetVer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPillowCVer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTowelVer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>If Verified</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtVerify">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SUBMIT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
